Framework/script/RepoCleaner/o2-qc-repo-move-objects

```diff
@@ -0,0 +1,99 @@
+#!/usr/bin/env python3
+
+import logging
+import argparse
+import dryable
+from Ccdb import Ccdb
+import sys
+import re
+
+def parseArgs():
+    """Parse the arguments passed to the script."""
+    logging.info("Parsing arguments")
+    parser = argparse.ArgumentParser(description='Move all objects in path to new-path')
+    parser.add_argument('--url', dest='url', action='store', help='URL of the CCDB, with http[s]://', required=True)
+    parser.add_argument('--log-level', dest='log_level', action='store', default="20",
+                        help='Log level (CRITICAL->50, ERROR->40, WARNING->30, INFO->20,DEBUG->10)')
+    parser.add_argument('--dry-run', action='store_true',
+                        help='Dry run, no actual deletion nor modification to the CCDB.')
+    parser.add_argument('--path', dest='path', action='store', default="",
```

Comment:
can you add an example of such a path? So that we do not add `/.*` because it is added automatically.

Reply:
agreed and done